webpack.config.ts

```diff
@@ -1,45 +1,52 @@
 /// <reference types="./source/globals" />
 
 import path from 'path';
-import {readdirSync, readFileSync} from 'fs';
+import {readFileSync} from 'fs';
 
 import SizePlugin from 'size-plugin';
-import stripIndent from 'strip-indent';
 // @ts-expect-error
 import {ESBuildPlugin} from 'esbuild-loader';
 import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
 
-function parseFeatureDetails(id: FeatureID): FeatureMeta {
-	const content = readFileSync(`source/features/${id}.tsx`, {encoding: 'utf-8'});
-	const fields = ['disabled', 'description', 'screenshot'] as const;
+const readmeContent = readFileSync(path.join(__dirname, 'readme.md'), 'utf-8');
```

Comment:
The problem with this is that it doesn't update when the file updates. What you can do is throttle it so it's only read once every build:

```suggestion
const getReadmeContent = debounce(() => readFileSync(path.join(__dirname, 'readme.md'), 'utf-8'), {
  delay: 500
});
```

We use `debounce-fn` for this